doc/Language/nativecall.pod6

```diff
@@ -545,6 +545,58 @@ for 1 ..^ $n {
 }
 =end code
 
+=head1 Strings
+
+Let's say there is some C code that caches strings passed, like so:
+
+=begin code :skip-test
```

Comment:
I think you can add `:lang<C>` here and it will syntax highlight it correctly.

Reply:
Updated